content/_components/onedrive/index.md

```diff
@@ -6,152 +6,158 @@ description: This is the component for working with Microsoft OneDrive storage s
+  * **Upsert** - If folder already exist, you will get information about this folder
 
-```
-{
-    "path": "base_folder/inner_folder/file.any"
-}
-```
+#### Input Metadata
 
-### Upsert File
+* **Path** - (string, required): Full path to folder where you need to create new folder, ex: `Monthly reports/November`
+* **Name** - (string, required): Name of new folder
 
-Action upserts (create or replace) with first file from attachment by provided path in Microsoft One Drive
+#### Output Metadata
 
-![Upsert File](img/upsert-file.png)
+Metadata of created folder
 
-#### Input fields description
+### Delete File
 
-* **Drive Identity** - OneDrive instance to work with. Selects by owner
+Action to delete item from OneDrive by provided path in selected disc.
 
-#### Metadata fields description
+#### Configuration Fields
 
-* **Path** - Full path to item to create or replace
+* **Drive Identity** - (dropdown, required): OneDrive instance to work with
+* **Emit strategy when file not found** - (dropdown, optional, `Emit nothing` by default) - select one of options to handle case when file not exist:
+  * **Emit nothing** - component will not produce any messages
+  * **Emit an empty object** - result will be empty object: `{}`
+  * **Throw an error** - error will be thrown
 
-#### Input example:
+#### Input Metadata
 
-```
-{
-    "path": "base_folder/inner_folder/file.any"
-}
-```
+* **Path** - (string, required): Full path to item, ex: `Monthly reports/November/Cars sales.pdf`
 
-### Delete File
+#### Output Metadata
 
-Action to delete item from OneDrive by provided path in selected disc.
-Returns filename if file was deleted and empty message if already wasn't exist
+* **Path** - (string, required): Full path to item, ex: `Monthly reports/November/Cars sales.pdf`
 
-![Delete File](img/delete-file.png)
+### Get File
 
-#### Input fields description
+Lookup a single file by its path.
 
-* **Drive Identity** - OneDrive instance to work with. Selects by owner
+#### Configuration Fields
 
-#### Metadata fields description
+* **Drive Identity** - (dropdown, required): OneDrive instance to work with
+* **Enable File Attachments** - (checkbox, optional): If checked, file will be uploaded to local storage and link provided in response
 
-* **Path** - Full path to item to delete
+#### Input Metadata
 
-#### Input example:
+* **Path** - (string, required): Full path to item, ex: `Monthly reports/November/Cars sales.pdf`
 
-```
-{
-    "path": "base_folder/inner_folder/file.any"
-}
-```
+#### Output Metadata
 
-### Create Folder
+File information as JSON object, if `Enable File Attachments` checked, there also will be additional field `attachmentUrl` with link to file on platform
```

Comment:
File information as JSON object, if `Enable File Attachments` is checked, there also will be an additional field `attachmentUrl` with a link to the file on the platform